modules/ROOT/pages/clustering/servers.adoc

```diff
@@ -37,8 +37,19 @@ When first discovered, a server's name defaults to the value of its generated se
 [[server-enabled-state]]
 === Enabled state
 
-A server in the free state needs to be explicitly enabled in order to be considered an active member of the cluster.
-The command `ENABLE SERVER server name` is used to transition a server into this _Enabled_ state.
+When you deploy a cluster for the first time, all servers included in the initial deployment are automatically enabled.
+For details, refer to the example on how to xref:clustering/setup/deploy.adoc#cluster-example-configure-a-three-primary-cluster[Configure a cluster with three servers].
+
+If you add a new server after the cluster is already running, the server is added in the `FREE` state.
+A server in the `FREE` state needs to be explicitly enabled in order to be considered an active member of the cluster.
+
+To transition a server into the _Enabled_ state, use the command `ENABLE SERVER`:
+
+[source, syntax, role=noheader]
+----
+ENABLE SERVER 'serverId' [OPTIONS "{" option: value[,...] "}"]
```

Comment:
What should we use here: 'serverId' or 'name'?
The next sentence says that the server's initial name is its ID. But we don't mention its name above.